### DISCUSSION TITLE
Laravel 9

### DISCUSSION THREAD

**Discussion by warmwhisky** [rappasoft/laravel-boilerplate#1571] [Q&A]
Are you planning or working on a version with Laravel 9?

I use this repo on an app with Laravel 6 and have been anticipating the upgrade to LTS for a while. I tried out upgrading to 9 yesterday and am wondering if I should wait for Laravel 9 to propagate amongst developers and settle?

**Comment by howdu** [comment]
Likely waiting for other packages to be updated https://github.com/langleyfoxall/laravel-nist-password-rules/issues/47

**Comment by Magentron** [comment]
No response yet from the maintainer for more than a year, is the project still alive?
Please let us know if so @rappasoft , so we can adjust accordingly.

**Comment by rappasoft** [comment]
Do people still use this project? I've been working on a package that installs an admin area onto Jetstream that will probably replace this. I can update this when I have time if there's still demand, but it will be stuck on BS4.

**Reply by calvez** [reply]
I made 2-3 apps based on this project, and the domain driven architecture and the wisely separated events, services etc amazed me. So yes, I m still using and it's a key point for me for best practices. On my next projects I have to use Laravel 10 so I would be glad if you recommend me your replacement. I know there are  Starter Kits as Breeze and Jetstream but somehow I liked your approach and solutions very much.

**Reply by bryanrscott** [reply]
We still use as well.

**Reply by ez-it-solutions** [reply]
I tried to update this myself but I'm pretty new to Laravel and ran into problems when forcing all packages to update to the latest packages. Everything seemed to work except for some of the front-end code.

I would love to have an updated version of this!!

**Comment by Magentron** [comment]
Hi Anthony,

Yes we're still using this project as the go-get starter project for starting a Laravel project.
If you think there are better options please share it.
Thanks BTW for this project, it has been a great tool for us! 👍

**Comment by hexpit** [comment]
I am currently working over TALL Stack based version of this boilerplate, it uses Laravel 10.9
Repository: https://github.com/developervijay7/laravel-quickstart
Website: https://laravel-quickstart.com

**Reply by calvez** [reply]
awesome

**Comment by WMspace** [comment]
I can only add that your work is very important, because it is a good example of how to write on Laravel. I'm really looking forward to the update too! **Would it make sense to start with Jetstream 4, Filament 3 Template?**
Best regards.

**Comment by ORN-Fox** [comment]
I love this project that I use as a basis for five of my projects (since version 5.2.2) to have a solid, clean and well-framed base. I'm sad to see it left to live without a recent update. Your work is important and of very good quality for all of us! ❤️

**Comment by DimuthuPHD** [comment]
I've utilized this package, and it's been fantastic. I hope future updates will be compatible with Laravel 10.  ❤️

**Comment by DimuthuPHD** [comment]
Any luck on the upgrade?

**Comment by warmwhisky** [comment]
I decided to start afresh with a Laravel installation and used Jetstream for authentication. This has been my preferred approach for all the applications I've worked on since I first posted about it.
